### PR TITLE
Fix issue template yaml syntax

### PR DESCRIPTION
**Pull request by sebsoto**
Unquoted '@' was causing a yaml parsing error.
Additionally the assignees field was explicitly made an array.

### CODE REVIEW

**Comment by sebsoto** [comment]
How this actually renders can be checked by clicking "view file" in the `Files changed` tab
example https://github.com/openshift/windows-machine-config-operator/blob/22c27cec36fe4b16479bab9db8cb1ae372741e2a/.github/ISSUE_TEMPLATE/bug.yml

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -2,14 +2,15 @@ name: Bug Report
...
+  - "@openshift/openshift-team-windows-containers"
 body:
   - type: markdown
     attributes:
       value: |
         Thanks for taking the time to fill out this bug report! Before filling out this form,
-        please review the (Cluster and OS Pre-Requisites)[https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md],
-        as well as the (BYOH Instance Pre-requisites)[BYOH Instance Pre-requisites].
+        please review the [Cluster and OS Pre-Requisites](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md),
+        as well as the [BYOH Instance Pre-requisites](BYOH Instance Pre-requisites).
```
Comment (aravindhp) [inline .github/ISSUE_TEMPLATE/bug.yml:13]: Now that I am able to see the rendered yaml, I suggest replacing the `Community issue` label with something like `Attention` or `Heads up`.  WDYT?

**Comment by sebsoto** [comment]
/hold
to allow higher priority PR to merge

**Review by aravindhp — APPROVED** [review]

**Review by aravindhp — APPROVED** [review]

**Pull request merged**